packages/js-sdk/src/InstantDistributionAgreementV1Helper.d.ts

```diff
@@ -3,11 +3,144 @@ import type { Framework } from "./Framework";
 import type { LoadedContract } from "./loadContracts";
 import type BN from 'bn.js';
 
-export interface Subscription {
+// comes from getIndex in IDAv1 contract,
+// then passed into _sanitizeIndexData
+// then returned from getIndex
+export interface IndexData {
     exist: boolean;
     approved: boolean;
-    units: BN;
-    pendingDistribution: BN;
+    totalUnitsApproved: string;
+    totalUnitsPending: string;
+}
+
+// comes from getSubscription in IDAv1 contract,
+// then passed into _sanitizeSubscriptionData
+// then returned from getSubscription
+export interface SubscriptionData {
+    exist: boolean;
+    approved: boolean;
+    units: string;
+    pendingDistribution: string
+}
+
+// comes from listSubscriptions in IDAv1 contract,
+// then passed into __sanitizeSubscriptionInfo
+// then returned from listSubscriptions
+export type SubscriptionInfoList = {
+    publisher: string;
+    indexId: string;
```

Comment:
`indexId` should be of type number (L570 of `InstantDistributionAgreementV1Helper.js`).